lib/xmlrpc/config.rb

```diff
@@ -33,6 +33,27 @@ module Config
     # enable Introspection extension by default
     ENABLE_INTROSPECTION = false
 
+    [
+      :DEFAULT_WRITER,
+      :DEFAULT_PARSER,
+      :ENABLE_NIL_CREATE,
+      :ENABLE_NIL_PARSER,
+      :ENABLE_BIGINT,
+      :ENABLE_MARSHALLING,
+      :ENABLE_MULTICALL,
+      :ENABLE_INTROSPECTION
+    ].each do |option|
+      define_singleton_method("#{option.downcase}") do
```

Comment:
Could you append `?` only for `ENABLE_*`?
For example, `enable_nil_create?` for `:ENABLE_NIL_CREATE`.

Reply:
Thanks.
But do we need both of `enable_nil_create` and `enable_nil_create?`?